### ISSUE TITLE
Windows 11 nvim Installation

### ISSUE THREAD

**Issue by mayorbyrne** [quick-lint/quick-lint-js#1021]
I installed neovim using winget and I just went through the instructions for installing quick-lint-js, and had a little bit of a rough time.

In this instructions, it says to
`Create the directory %USERPROFILE%\vimfiles\pack\external\start\`

and place the extracted `quick-lint-js.vim` inside.

Doing so gives me some errors on start:
```Error detected while processing C:\Users\Kevin\AppData\Local\nvim\init.lua:
E5113: Error while calling lua chunk: C:\Users\Kevin\AppData\Local\nvim\init.lua:1: module 'lspconfig/quick_lint_js' not found:
        no field package.preload['lspconfig/quick_lint_js']
        no file '.\lspconfig/quick_lint_js.lua'
        no file 'C:\Users\Kevin\AppData\Local\Programs\Neovim\bin\lua\lspconfig/quick_lint_js.lua'
        no file 'C:\Users\Kevin\AppData\Local\Programs\Neovim\bin\lua\lspconfig/quick_lint_js\init.lua'
        no file '.\lspconfig/quick_lint_js.dll'
        no file 'C:\Users\Kevin\AppData\Local\Programs\Neovim\bin\lspconfig/quick_lint_js.dll'
        no file 'C:\Users\Kevin\AppData\Local\Programs\Neovim\bin\loadall.dll'
stack traceback:
        [C]: in function 'require'
```
        
The only way I was able to get it working was to copy the _contents_ of the `quick-lint-js.vim` directory and place it inside the 
`%LocalAppData%/Programs/Neovim/bin` directory.

Am I doing something wrong, or do the instructions need tweaking a bit?

**Comment by strager** [comment]
I think the documentation is wrong. It's telling you the path for Vim, not Neovim.

https://github.com/quick-lint/quick-lint-js/blob/3030a20f0004f4559f03c2892d30ffd90427f26e/website/public/install/neovim/common-manual.ejs.html#L26

I think `%LocalAppData%/Programs/Neovim/bin` is also the wrong path (because it depends on where Neovim is installed). I need to install Neovim on Windows myself and see where the user config goes.

**Comment by jaitjacob** [comment]
> ... I need to install Neovim on Windows myself and see where the user config goes.

@strager my installation of Neovim has user config located here `C:\Users\{{my-windows-username}}\AppData\Local\nvim\`

**Comment by mayorbyrne** [comment]
> > ... I need to install Neovim on Windows myself and see where the user config goes.
> 
> @strager my installation of Neovim has user config located here `C:\Users\{{my-windows-username}}\AppData\Local\nvim\`

Yes, this is where mine resides too. I forgot to mention that in the original post.

**Comment by strager** [comment]
I fixed the docs in commit 2536fd7123707a80aa0d9bf8f680f329a478041b. Thanks for reporting!
